Currently not able to get the tests to pass. If I run in a local 3.12 venv as a standalone, i.e. clone repo, install reqs, run pytest I get the following error:
```
FAILED tests/test_ingest.py::test_htsget_ingest - requests.exceptions.MissingSchema: Invalid URL '/genomics/ga4gh/drs/v1/objects/HG00096.cnv.vcf': No scheme supplied. Perhaps you meant https:///genomics/ga4gh/drs/v1/object...
```

run pytest within the ingest container of my partially running stack after updating the branch and rebuilding I get
```
# pytest
============================================================================================================== test session starts ==============================================================================================================
platform linux -- Python 3.10.13, pytest-7.2.0, pluggy-1.3.0
rootdir: /ingest_app
plugins: requests-mock-1.11.0
collected 8 items / 1 error                                                                                                                                                                                                                     

==================================================================================================================== ERRORS =====================================================================================================================
_____________________________________________________________________________________________________ ERROR collecting tests/test_ingest.py _____________________________________________________________________________________________________
ImportError while importing test module '/ingest_app/tests/test_ingest.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_ingest.py:7: in <module>
    import katsu_ingest
E   ModuleNotFoundError: No module named 'katsu_ingest'
=============================================================================================================== warnings summary ================================================================================================================
clinical_ETL_code/CSVConvert.py:319
clinical_ETL_code/CSVConvert.py:319
clinical_ETL_code/CSVConvert.py:319
clinical_ETL_code/CSVConvert.py:319
  /ingest_app/clinical_ETL_code/CSVConvert.py:319: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    .applymap(str) \

clinical_ETL_code/CSVConvert.py:320
clinical_ETL_code/CSVConvert.py:320
clinical_ETL_code/CSVConvert.py:320
clinical_ETL_code/CSVConvert.py:320
  /ingest_app/clinical_ETL_code/CSVConvert.py:320: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    .applymap(lambda x: x.strip()) \

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================================================================================ short test summary info ============================================================================================================
ERROR tests/test_ingest.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================================================= 8 warnings, 1 error in 4.09s ==========================================================================================================
# 
```

entirely possible (and likely) I am doing something wrong though

--- a/README.md
+++ b/README.md
@@ -20,8 +20,6 @@ Using a Python 3.10+ environment, run the following:
 
 ```bash
 pip install -r requirements.txt
-git submodule update --init --recursive
-pip install -r clinical_ETL_code/requirements.txt
 ```
 
 ### Set environment variables

--- a/tests/test_ingest.py
+++ b/tests/test_ingest.py
@@ -3,7 +3,10 @@ import json
 import requests
 import os
 import re
+import sys
 
+REPO_DIR = os.path.abspath(f"{os.path.dirname(os.path.realpath(__file__))}/..")
+sys.path.insert(0, os.path.abspath(f"{REPO_DIR}"))
 import katsu_ingest
 import htsget_ingest
 

